main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+	"siteapi/app"
+	"siteapi/app/database"
+)
+
+func main() {
+	app := app.New()
+	app.DB = &database.DB{}
+	err := app.DB.Open()
+	check(err)
```

Comment:
Aqui ao invés de manualmente logar/dar o exit(1), tu pode simplesmente:
```suggestion
	panic(err)
```